openerp/addons/base/ir/ir_ui_view.py

```diff
@@ -232,15 +232,13 @@ def graph_get(self, cr, uid, id, model, node_obj, conn_obj, src_node, des_node,
                     if model_value._obj==node_obj:
                         _Node_Field=model_key
                         _Model_Field=model_value._fields_id
-                    flag=False
                     for node_key,node_value in _Node_Obj._columns.items():
                         if node_value._type=='one2many':
                              if node_value._obj==conn_obj:
-                                 if src_node in _Arrow_Obj._columns and flag:
+                                 if des_node in _Arrow_Obj._columns and node_value._fields_id == des_node:
```

Comment:
The tests for the presence of `src_node` and `des_node` in _Arrow_Obj._columns is redundant, if they match the `_fields_id` they are always in the columns. I'll remove it when merging, and add a comment to document the weird `des_node -> _Source_Field` and `src_node -> _Destination_Field` relationship.